funnel/views/commentvote.py

```diff
@@ -125,6 +125,8 @@ def new_comment(self):
                 if item.get('to'):
                     # Sender is set to None to prevent revealing email.
                     send_mail(sender=None, body=email_body, **item)
+        else:
+            flash(_("Something went wrong, please try again"), category='error')
```

Comment:
Errors are idempotent. If something went wrong, it will go wrong again. This needs a more accurate error message.

Reply:
This is validation error. Comment form can't show validation error right now. It needs to be redone to show these. I'll check if it can be done with this PR.

Reply:
We should render the errors via the flash message for now, but the correct fix is to turn this into an Ajax call.